sensor_msgs/CompressedImage imgL
sensor_msgs/CompressedImage imgR
bool isOkay
